### DISCUSSION TITLE
Published Website

### DISCUSSION THREAD

**Discussion by joshzcold** [joshzcold/Cold-Family-Feud#32] [Announcements]
Cold-Family-Feud is now available at https://famf.app !

It is currently using the free tier of heroku so if you are accessing it then you might have to wait a second for the app to become un-suspended before it loads up.

Changes: 
- websockets run through secure `wss` and the local dev envrionment has some self signed certs to make them functional
- new default games `season20` (general questions from the show) and `tech` (custom game featuring tech questions)
- new script `scripts/point_generator` which can generate realistic point values givin a question amount along with min/max point values. _requires ruby_

This is useful when creating your own questions and you don't want to come up with point values off the top of your head.

![image](https://user-images.githubusercontent.com/36175703/126075725-f3fd0a42-d41e-4a70-856b-14d26bbf9c83.png)

- game creator now allows skipping creating the fast money round.
- created license (MIT) 
- websockets get served at a path instead of a dedicated port. This is to allow the websockets to function in production

Getting the website published was a painful process, but now that its finished im excited for the potential of non-tech individuals being able to use the project. For instance my mother in law already want to try and use it for her classroom. I think thats pretty cool 😀.

I currently don't know the limits of the free tier on heroku when used against this project. Hopefully its enough for the foreseeable future as people stumble upon this project and try a quick game. 

For the next releases ill be focusing on play-ability improvements using mobile. Off the bat I would like if you didn't have to screen share the game window to be able to play the game. 

<hr /><em>This discussion was created from the release <a href='https://github.com/joshzcold/Cold-Family-Feud/releases/tag/1.3.1'>Published Website</a>.</em>

**Comment by joshzcold** [comment]
As an update on this. I played the game with my work. 
About 30 players. 

- No crashes 👍
- We were able to finish the game w/ fast money 👍
- on famf.app production site websockets didn't keep alive so we had to keep refreshing our browser 👎
   tracking issue: https://github.com/joshzcold/Cold-Family-Feud/issues/33

**Comment by pbkamill** [comment]
Thank you very much for this project! 👍 

My family loves the Family Feud, and this project made it possible to have a fantastic birthday party! 

I appreciate your effort!

**Reply by joshzcold** [reply]
Thank you! I appreciate it ♥️

I'm glad you had a great birthday!

**Comment by OsakaBoi** [comment]
Also wanted to express my appreciation for your work. Just hosted a virtual office Christmas family feud. Was happy to see 3 Christmas themed question sets available. The website worked smoothly

**Reply by joshzcold** [reply]
Thank you 🎅
Love to hear it!
